### PR TITLE
ign -> gz Upstream Macro Migration : gz-msgs

### PR DESCRIPTION
**Pull request by methylDragon**
See: https://github.com/gazebo-tooling/release-tools/issues/737

### CODE REVIEW

**Review by chapulina — APPROVED** [review]
LGTM once upstream is merged.

File: src/ign.hh
Comment (chapulina) [inline src/ign.hh:34]: Oh this header isn't installed (it isn't in the `include` directory), so we can do a hard tock

Comment (chapulina) [reply src/ign.hh]: [d67ca90](https://github.com/gazebosim/gz-msgs/pull/260/commits/d67ca90169437fe87af6876f6aa60d856596281c)

**Review by chapulina — APPROVED** [review]

**Comment by methylDragon** [comment]
@osrf-jenkins run tests please!

**Pull request merged**
